packages/atlas-service/src/main.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from './util';
 import {
   broadcast,
-  getStoragePaths,
+  getAppName,
```

Comment:
I removed usage of this method completely from atlas-service, just FYI, you probably can resolve conflicts here by reverting this change